### PR TITLE
Add information how to test action flash messages

### PR DESCRIPTION
**Pull request by davydovanton**
/cc @cllns

### CODE REVIEW

File: source/guides/actions/testing.md [hunk 1]
```diff
@@ -215,6 +215,31 @@ end
...
 
+
+### Flash messages
+
+In your action tests, you can check `flash` messages too. For this, you can use `exposures` method for getting all flash data.
+
+The following test example uses this method.
+
+```ruby
+# spec/web/controllers/users/show_spec.rb
```
Comment (AlfonsoUceda) [inline source/guides/actions/testing.md]: s/show_spec/create_spec

**Comment by cllns** [comment]
This would've helped me last week :)

However, we don't mention `flash` anywhere else. It's experimental so we don't have it in the guides. Personally, I think it's time we add it to the guides with a warning. Don't think this should go in before that though. 💞 

Also, we should add a `sessions` testing guide, since it's extremely related: http://hanamirb.org/guides/actions/sessions/

**Comment by AlfonsoUceda** [comment]
@jodosha I think we should remove experimental flag from flash, what do you think?

**Comment by davydovanton** [comment]
agree with @AlfonsoUceda 👍

**Comment by davydovanton** [comment]
merged, thanks for review @jodosha  👍

**Pull request merged**